dwave/system/composites/directembedding.py

```diff
@@ -0,0 +1,277 @@
+# coding: utf-8
+# Copyright 2020 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# =============================================================================
+"""Composites that directly embed small problems defined on qubit indices onto
+a QPU's working graph.
+"""
+import dwave_networkx as dnx
+import dimod
+from dimod.exceptions import BinaryQuadraticModelStructureError
+from dwave_networkx import chimera_graph, pegasus_graph
+from dwave.system.composites.embedding import FixedEmbeddingComposite
+from dwave.embedding.exceptions import EmbeddingError
+
+__all__ = ('DirectChimeraTilesEmbeddingComposite',
+          )
+
+class DirectChimeraTilesEmbeddingComposite(dimod.ComposedSampler):
```

Comment:
Should this not be a [StructuredSampler](https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/sampler_composites/api.html#dimod.Structured)?

Reply:
I think so